doubly_linked_list/doubly_linked_list.py

```diff
@@ -1,68 +1,170 @@
 """Each ListNode holds a reference to its previous node
 as well as its next node in the List."""
+
+
 class ListNode:
-  def __init__(self, value, prev=None, next=None):
-    self.value = value
-    self.prev = prev
-    self.next = next
+    def __init__(self, value, prev=None, next=None):
+        self.value = value
+        self.prev = prev
+        self.next = next
 
-  """Wrap the given value in a ListNode and insert it
+    """Wrap the given value in a ListNode and insert it
   after this node. Note that this node could already
   have a next node it is point to."""
-  def insert_after(self, value):
-    current_next = self.next
-    self.next = ListNode(value, self, current_next)
-    if current_next:
-      current_next.prev = self.next
 
-  """Wrap the given value in a ListNode and insert it
+    def insert_after(self, value):
+        current_next = self.next
+        self.next = ListNode(value, self, current_next)
+        if current_next:
+            current_next.prev = self.next
```

Comment:
Weird right? It's so hard to picture this in your head, but if you draw it out, it's pretty simple. Your sprint challenge will pertain to this, definitely!